apps/playground/src/app/modules/components/toggle-module/toggle-module.component.spec.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Angular imports
+ */
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+/**
+ * Internal imports
+ */
+import { ToggleModuleComponent } from './toggle-module.component';
+
+describe('ToggleModuleComponent', () => {
+  let component: ToggleModuleComponent;
+  let fixture: ComponentFixture<ToggleModuleComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ToggleModuleComponent]
```

Comment:
The test passes but there are mistakes in the output.

You can replace this line by this : `imports: [NoopAnimationsModule, ModulesModule]`